### DISCUSSION TITLE
json syntax error

### DISCUSSION THREAD

**Discussion by ebolisa** [micropython/micropython#9238] [Using MicroPython] [answered]
Hi,

I cannot seem to find the problem with this code. However, it works on my PC.
I appreciate some help.

```
{
"ssid": "PHONE",
"pw": "Jar",
"ftpServer": "online.net",
"url_username" = "user",
"url_pass" = "pass"
}
```

```
# read from file
def r_json(key):
    try:
        with open(json_file, 'rb') as f:
            data = ujson.load(f)
            return data[key]
    except Exception as e:
        print(e)
        return 0.0

print(r_json("ssid"))
```

ERROR:

```
syntax error in JSON
0.0
```

eMicroPython v1.19.1 on 2022-06-18; ESP32 module with ESP32
also fails on MicroPython v1.19.1 on 2022-09-06; Raspberry Pi Pico with RP2040

**Comment by cwalther** [comment, marked as answer]
There are some `=` in your JSON that should be `:`.

**Reply by ebolisa** [reply]
Thank you! I don't know how many times gone over that file without noticing it.
